src/ape/types/__init__.py

```diff
@@ -264,6 +267,19 @@ def get(self, item: str, default: Optional[Any] = None) -> Any:
         return self.event_arguments.get(item, default)
 
 
+class ContractLogContainer(list):
+    __test__ = False
+
+    def filter(self, event, **kwargs) -> List[ContractLog]:
+        found_events = []
+        for log in self:
+            if log.event_name == event.name:
+                match = all(v == log.event_arguments.get(k, _NONE) for k, v in kwargs.items())
```

Comment:
Wouldn't this work the same? 
```suggestion
                match = all(v == log.event_arguments.get(k, None) for k, v in kwargs.items())
```

Reply:
`None` is the default so you wouldn't need to specify, but ya wondering why we need the `_None` object as well.

Reply:
as @istankovic explained, if we have expression like .filter(event, foo=None) which foo doesnt exist in event then log.event_arguments.get("foo", None) == None which would be true and we could see some unwanted things in response. using _None makes sure the log wont be there.

Reply:
Can just assert that the value×in the kwarg is not `None`

Reply:
also works so changed